packages/samples/headless-react/src/components/date-filter/date-filter.class.tsx

```diff
@@ -0,0 +1,93 @@
+import {Component, ContextType} from 'react';
```

Comment:
We'll need to add entries for the date and numeric filter controllers in `packages\headless\doc-parser\use-cases\search.ts` and reference these files, and then they should appear in the generated docs 🤞